CONTRIBUTING.md

```diff
@@ -61,7 +61,7 @@ through the commit history. It also automates [semantic versioning](http://semve
 
 - Explain the additions/edits/fixes made in your staged changes. If you cannot describe it within
   ~50 characters, you should be breaking it into multiple commits
-- Use the imperative mood (e.g. "fixed", not "fix")
+- Use the imperative mood (e.g. "fix", not "fixed")
```

Comment:
I'm not familiar with imperative mood, but it seems this change supports the wiki definition. From the conventional commit format specification:

```
feat(parser): add ability to parse arrays // command
vs
feat(parser): added ability to parse arrays // past-tense action
```

I've done the latter as a description of work done where the former matches what might appear on a task board. Was this something noticed during work to update module URLs?

Reply:
Yea looks like this is a typo we never caught. This should be changed.